book/src/integrations/pam_and_nsswitch.md

```diff
@@ -38,11 +38,6 @@ You can check the privileged tasks daemon is running with:
 systemctl status kanidm-unixd-tasks
 ```
 
-> [!NOTE]
->
-> The `kanidm_unixd_tasks` daemon is not required for PAM and nsswitch functionality. If disabled, your system will
-> function as usual. It is however strongly recommended due to the features it provides.
-
```

Comment:
I'd make this a "you *must* have it now, and this is a change from our original design"